ui/apps/platform/cypress/integration/integrations/integrations.helpers.js

```diff
@@ -430,7 +430,8 @@ export function generateCreatedAuthProvidersIntegrationInForm(
             cy.get(selectors.buttons.generate).click();
         },
         routeMatcherMap,
-        staticResponseMap
+        staticResponseMap,
+        { timeout: 20000 }
```

Comment:
@ivan-degtiarenko Two items regarding tests:
1. The majority of the ui-e2e failures can be fixed with a rebase. There were some flaky/persistent failing tests that were fixed two days ago
2. I think the API token failure is legit though, as it looks like the backend is not allowing an unset API token `""` and is failing the generation request. According to previous comments, I believe the backend should accept `""` and use a default date when no value is provided. There is evidence of this in the Cypress video output for a recent failed job: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/stackrox_stackrox/7914/pull-ci-stackrox-stackrox-master-gke-ui-e2e-tests/1762800750028656640/artifacts/gke-ui-e2e-tests/stackrox-stackrox-e2e-test/artifacts/videos/integrations/apiTokens.test.js.mp4

Reply:
@dvail Regarding 2: you're right, I was testing locally against the wrong backend 🤦 . 

API right now works like this:
1. If expiration is set to anything, including `""`, then it is expected to be parseable into time
2. If expiration is not set, then it is treated as null value and default value of 1 year is used
I suggest we just don't send anything from the FE instead of sending `""` as expiration - WDYT?

Reply:
Sounds good to me, and with the recent changes sending `""` shouldn't even be possible anyway. 👍